Samples/VirtualCamera/VirtualCameraManager_WinRT/Constants.h

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+#pragma once
+#include "guiddef.h"
+
+namespace winrt::VirtualCameraManager_WinRT::implementation
+{
+    // Define virtual camera CLSID (refer to virtual camera dll, should match the CLSID of the )
+    static const GUID CLSID_SIMPLEMEDIASOURCE_WIN32 = { 0x7b89b92e, 0xfe71, 0x42d0, {0x8a, 0x41, 0xe1, 0x37, 0xd0, 0x6e, 0xa1, 0x84 } };

```

Comment:
CLSDI_SIMPLEMEDIASOIURCE_WIN32 is updated to CLSID_VirtualCameraMediaSource 
and below name is changed too, should update here.